metrics/prometheus-agent/templates/prometheus.yaml

```diff
@@ -29,6 +29,8 @@ spec:
     - --web.console.libraries=/etc/prometheus/console_libraries
     - --web.route-prefix=/
     - --web.config.file=/etc/prometheus/web_config/web-config.yaml
+    - --log.level={{ .Values.prometheus.prometheusSpec.logLevel }}
+    - --log.format={{ .Values.prometheus.prometheusSpec.logFormat }}
```

Comment:
Should default to json (I would even set it up as json hardcoded)

Reply:
This is json by default and also warn by default, but since it's a public chart I prefer to let the option to change available.